imgui-examples/examples/support/mod.rs

```diff
@@ -111,6 +113,46 @@ pub fn init(title: &str) -> System {
     }
 }
 
+struct ViewportStorage<'a, T: 'static> {
+    event_loop: &'a glium::glutin::event_loop::EventLoopWindowTarget<T>,
+    viewports: &'a mut HashMap<imgui::Id, glium::glutin::window::Window>,
+}
+
+impl<'a, T> imgui_winit_support::WinitPlatformViewportStorage for ViewportStorage<'a, T> {
+    fn create_window(&mut self, id: imgui::Id, flags: imgui::ViewportFlags) {
+        let builder = WindowBuilder::new()
+            .with_always_on_top(flags.contains(ViewportFlags::TOP_MOST))
+            // .with_decorations(!flags.contains(ViewportFlags::NO_DECORATION))
```

Comment:
Seems like this should be uncommented

Reply:
Yep, i just commented it out as without any rendering code, the undecorated windows would be invisible and it would have been hard to test...